### PR TITLE
Fixes in change log for 2023.1

### PR DESCRIPTION
**Pull request by CyrilleB79**
Preliminary note: This PR only fixes small typo in the change log. Since 2023.1 translation freeze has already started, it is not worth merging it. However in case a new translation freeze is decided for 2023.1 it could be worth merging it.

### Link to issue number:
None
### Summary of the issue:
Small typos found while reviewing the translation of the change log.
### Description of user facing changes
See updates of the change log in this PR.
### Description of development approach
N/A
### Testing strategy:
Check the file generated by appVeyor.
### Known issues with pull request:
Should only be merged if a new translation freeze is decided.
### Change log entries:
N/A

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -15,9 +15,10 @@ Press ``NVDA+d`` to cycle through summaries when annotations are reported (e.g.
 Tivomatic Caiku Albatross 46/80 braille displays are now supported.
 
 Support for ARM64 and AMD64 versions of Windows has improved.
+
 There are many bug fixes, notably Windows 11 fixes.
 
-eSpeak, LibLouis, Sonic rate boost and Unicode CLDR has been updated.
+eSpeak, LibLouis, Sonic rate boost and Unicode CLDR have been updated.
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:21]: has -> have

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -15,9 +15,10 @@ Press ``NVDA+d`` to cycle through summaries when annotations are reported (e.g.
 Tivomatic Caiku Albatross 46/80 braille displays are now supported.
 
 Support for ARM64 and AMD64 versions of Windows has improved.
+
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:18]: Empty line to make two separate paragraphs

File: user_docs/en/changes.t2t [hunk 3]
```diff
@@ -95,7 +96,7 @@ Same as pressing ``NVDA+k`` twice, but may be more useful for braille users. (#1
 - Emojis should now be reported in more languages. (#14433)
 - The presence of an annotation is no longer missing in braille for some elements. (#13815)
 - Fixed an issue where config changes not save correctly when changing between a "Default" option and the value of the "Default" option. (#14133)
-- When configuring NVDA there will always be at least one key is defined as a NVDA key. (#14527)
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:98]: 1. "is" removed
2. "a" -> "an"

File: user_docs/en/changes.t2t [hunk 4]
```diff
@@ -109,7 +110,7 @@ Note: this is an Add-on API compatibility breaking release.
 Add-ons will need to be re-tested and have their manifest updated.
 Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documentation/developerGuide.html#API] for information on NVDA's API deprecation and removal process.
 
-- System test should now pass when run locally on non-English systems. (#13362)
+- System tests should now pass when run locally on non-English systems. (#13362)
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:113]: "test" -> "tests"

File: user_docs/en/changes.t2t [hunk 5]
```diff
@@ -154,7 +155,7 @@ Please open a GitHub issue if your Add-on has an issue with updating to the new
     - ``autoTether`` has been removed; ``tetherTo`` can now take the value "auto" instead.
     -
   - In ``[keyboard]`` section  (#14528):
-    - ``useCapsLockAsNVDAModifierKey``, ``useNumpadInsertAsNVDAModifierKey``, ``useExtendedInsertAsNVDAModifierKey`` has been removed.
+    - ``useCapsLockAsNVDAModifierKey``, ``useNumpadInsertAsNVDAModifierKey``, ``useExtendedInsertAsNVDAModifierKey`` have been removed.
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:158]: "has" -> "have"

**Comment by XLTechie** [comment]
@CyrilleB79 You might also want to get:

> Add-ons will need to be re-tested and have their manifest updated.

Many ad-ons do not share a manifest.

Manifest -> manifests

**Comment by CyrilleB79** [comment]
> @CyrilleB79 You might also want to get:
> Add-ons will need to be re-tested and have their manifest updated.
> Many ad-ons do not share a manifest. Manifest -> manifests

Thanks. I was unsure about this one and had trusted native English speakers.

**Comment by CyrilleB79** [comment]
Note: the failing system test is completely unrelated.

File: user_docs/en/changes.t2t [hunk 6]
```diff
@@ -106,10 +107,10 @@ Same as pressing ``NVDA+k`` twice, but may be more useful for braille users. (#1
 
 == Changes for Developers ==
 Note: this is an Add-on API compatibility breaking release.
-Add-ons will need to be re-tested and have their manifest updated.
+Add-ons will need to be re-tested and have their manifests updated.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:110]: I think this change has been discussed before, and either sentence is fine. The current grammar can be read as "the [add-ons should] have their manifest updated".

If this new string is preferred, we need to update a template elsewhere

**Review by seanbudd — APPROVED** [review]
thanks @CyrilleB79

**Comment by XLTechie** [comment]
Regarding the question of whether "manifest" should be plural or singular (i.e. 
do multiple add-ons share one manifest, or does manifest being singular agree 
with the pronoun "their"):

I think I did submit a change on this in either 2022 or 2021, but it wasn't 
accepted then for the same reason.

Here are a couple of questions on the subject, with interesting answers. My 
opinion seems to be the more common one in the English community, but there 
is a significant portion which holds @seanbudd's position as well.

https://www.englishforums.com/English/NounAfterTheir/brjhbr/post.htm

https://english.stackexchange.com/questions/551378/singular-or-plural-noun-after-their-when-subject-is-those-as-in-those-who

**Pull request merged**